contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol

```diff
@@ -16,12 +16,17 @@ contract HomeBridgeErcToNative is EternalStorage, BasicBridge, BasicHomeBridge,
     event AmountLimitExceeded(address recipient, uint256 value, bytes32 transactionHash);
 
     function () public payable {
+        nativeTransfer();
```

Comment:
Does it make sense to do the same for `HomeBridgeNativeToErc.sol`?

Reply:
Applied the same to `HomeBridgeNativeToErc` https://github.com/poanetwork/poa-bridge-contracts/pull/203/commits/17536e25d7e1c5cad848d55d4058a84a54d5700c